etc/run.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
+alembic upgrade head
 python load_csv.py --path ${DIALOG_DATA_PATH}
-uvicorn main:app --host 0.0.0.0 --port ${PORT}
+uvicorn main:app --host 0.0.0.0 --port ${PORT} --reload
```

Comment:
docker image is meant to run in production, not in a development environment, it doesn't make sense to `--reload` it!